app/javascript/components/widget/components/widget-pie-chart-legend/styles.scss

```diff
@@ -1,10 +1,17 @@
 .c-pie-chart-legend-widget {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
   padding-top: rem(30px);
 
-  .cover-legend {
-    min-width: 50%;
+  .pie-and-legend {
```

Comment:
I think this will break some cross browser views. Can you check internet explorer and safari 10.

Reply:
The min-width? It was there before 😬

Reply:
Ah you haven't changed the structure of the html. Ok all good.